tests/test_json.py

```diff
@@ -61,6 +61,7 @@ def test_args(path: Optional[Path]) -> None:
             "DebugSandbox": false,
             "DebugShell": false,
             "DebugWorkspace": false,
+            "DefaultConfig": {f'"{os.fspath(path)}"' if path is not None else "null"},
```

Comment:
Nit: I know this is only a test, but `Directory` and `DefaultConfig` having the same non-null value will never make sense. If somebody looks at this to see how something is used (I did in the past), this would be confusing, so maybe just hard-coding
```suggestion
            "DefaultConfig": null,
```
would be better.